src/colvarcomp_gpath.cpp

```diff
@@ -406,9 +402,8 @@ void colvar::gzpath::apply_force(colvarvalue const &force) {
 }
 
 colvar::linearCombination::linearCombination(std::string const &conf): cvc(conf) {
-    GeometricPathCV::init_string_cv_map(string_cv_map);
     // Lookup all available sub-cvcs
-    for (auto it_cv_map = string_cv_map.begin(); it_cv_map != string_cv_map.end(); ++it_cv_map) {
+    for (auto it_cv_map = colvar::get_global_cvc_map().begin(); it_cv_map != colvar::get_global_cvc_map().end(); ++it_cv_map) {
```

Comment:
@HanatoK Won't this line fail to compile if the C++ standard is too old and the ifdefs above are false?
Maybe we need to keep the old version of the map as a fallback.

@giacomofiorin Sorry if I'm late to the party here.

Reply:
@jhenin `colvarcomp_gpath.cpp` is conditionally compiled even in the old version, isn't it? I think the old version of the map also uses `std::function`, which is not available until C++11.

Reply:
It is. I checked a test build of the library files using both C++98 and C++11 before merging this.

Reply:
All clear. Thank you guys.